glossary.yml

```diff
@@ -3952,15 +4104,17 @@
   en:
     term: "XML"
     def: >
-      A set of rules for defining HTML-like tags and using them to format documents
-      (typically data). XML achieved license plate popularity in the early 2000s,
-      but its complexity led many programmers to adopt [JSON](#json) instead.
+      A set of rules for defining [HTML](#html)-like tags and using them to
+      format documents (typically data). XML achieved license plate popularity
```

Comment:
What is "license plate popularity"? I tried a Google search, but I couldn't figure out what this means.

Reply:
Slang for "popular enough to appear as a slogan on license plates" - I've changed it to match the Spanish "was popular in the early 2000s".